src/translation/german.c

```diff
@@ -119,7 +119,7 @@ static translation_string all_strings[] = {
     {TR_HOTKEY_ARROW_DOWN, "Runter"},
     {TR_HOTKEY_ARROW_LEFT, "Links"},
     {TR_HOTKEY_ARROW_RIGHT, "Rechts"},
-    {TR_HOTKEY_TOGGLE_FULLSCREEN, "Vollbild ein-/ausschalten"},
+    {TR_HOTKEY_TOGGLE_FULLSCREEN, "Vollbild ein-/ausschalten"}
```

Comment:
Missing a `,` at the end.

Reply:
Why is it missing when I didn't touch that?
Or did I by mistake? Like it was highlighted even though I wasn't looking for that part?